active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,335 @@
+
+| Value | Description |
+|:---:|:---|
+| PipsPagerItems | The collection of items to represent the pips in the PipsPager. |
+
+
+# API Details
+
+```IDL
+enum PipsPagerButtonVisibility
+{
+  Visible,
+  VisibleOnPointerOver,
+  Collapsed, 
+}
+
+runtimeclass PipsPagerSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+unsealed runtimeclass PipsPagerTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    Windows.Foundation.Collections.IVector<Object> PipsPagerItems { get; }; 
+}
+
+
+unsealed runtimeclass PipsPager : Windows.UI.Xaml.Controls.Control
+{
+    PipsPager();
+
+    [MUX_DEFAULT_VALUE("-1")]
+    Int32 NumberOfPages;
+
+    [MUX_DEFAULT_VALUE("0")]
+    Int32 SelectedPageIndex;
+
+    [MUX_DEFAULT_VALUE("5")]
+    Int32 MaxVisiblePips;
+
+    [MUX_DEFAULT_VALUE("Horizontal")]
+    Windows.UI.Xaml.Controls.Orientation Orientation;
+
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility PreviousButtonVisibility;    
+    [MUX_DEFAULT_VALUE("Collapsed")]
+    PagerControlButtonVisibility NextButtonVisibility;
+
+    Windows.UI.Xaml.Style PreviousButtonStyle;
+    Windows.UI.Xaml.Style NextButtonStyle; 
+
+    Windows.UI.Xaml.Style SelectedPipStyle;
+    Windows.UI.Xaml.Style DefaultPipStyle;
+
+
+    event Windows.Foundation.TypedEventHandler<PipsPager, PipsPagerSelectedIndexChangedEventArgs> SelectedIndexChanged;
+
+    PipsPagerTemplateSettings TemplateSettings{ get; };
+
+    static Windows.UI.Xaml.DependencyProperty NumberOfPagesProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedPageIndexProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty MaxVisiblePipsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty OrientationProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonVisibilityProperty { get; };
+    static Windows.UI.Xaml.DependencyProperty PreviousButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty NextButtonStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty SelectedPipStyleProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty DefaultPipStyleProperty{ get; };
+}
+```
+
+![](images/pager-control-vertical-pips-scroll-ex.jpg)
+
+
+# Input & Accessibility
+
+## Keyboard
+
+- Tab navigation into the PipsPager proceeds through the first actionable item. This would be the previous button (if not Collapsed) or the first pip. Any directional buttons can be navigated to via tab if they are not Collapsed and will be visible on focus.
+- The arrow keys can be used to navigate between the pips and directional buttons. Independent of the orientation set, the left and up arrow keys will change focus to the previous pip or to the previous button and the right and down arrow keys will change focus to the next pip or the next button.
```

Comment:
Presumably, navigating to a pip does not select it. (Otherwise entering the PipsPager would move you back 2 pages.) Does the space bar select the current navigated-to item?

Reply:
@oldnewthing navigating to a pip does not select it. Space bar will select the current navigated-to item.

Reply:
@gabbybilka: Tab navigation into the PipsPager proceeds through the first actionable item. This would be the previous button (if not Collapsed) or **CURRENTLY SELECTED PIP**. Any directional buttons can be navigated to via tab if they are not Collapsed and will be visible on focus.

Reply:
Added both space to select and currently selected pip note in latest update, thanks.